translations/ui-organizations/en.json

```diff
@@ -10,16 +10,16 @@
   "data.contactTypes.address": "Address",
   "data.contactTypes.addressLine1": "Address 1",
   "data.contactTypes.addressLine2": "Address 2",
-  "data.contactTypes.stateProviceOrRegion": "State/Province/Region",
-  "data.contactTypes.zipOrPostalCode": "Zip/Postal code",
+  "data.contactTypes.stateProviceOrRegion": "State/province/region",
+  "data.contactTypes.zipOrPostalCode": "Zip/postal code",
```

Comment:
the same, usually `Postal code` goes with capital

Reply:
Ok